### PR TITLE
3.3.20 Avoid seek after pause

### PR DESCRIPTION
**Pull request by emveepee**
No need to seek when there is buffer available

### CODE REVIEW

File: src/buffers/ClientTimeshift.cpp [hunk 1]
```diff
@@ -115,6 +115,24 @@ void ClientTimeShift::Close()
...
+void ClientTimeShift::Resume()
+{
+  ClientTimeShift::GetStreamInfo();
+  if (m_stream_duration > g_timeShiftBufferSeconds)
+  {
+    int64_t startSlipBuffer = m_stream_length - (g_timeShiftBufferSeconds * m_stream_length/m_stream_duration);
+    XBMC->Log(LOG_DEBUG, "%s:%d: %lld %lld %lld", __FUNCTION__, __LINE__, startSlipBuffer, m_streamPosition, m_stream_length.load());
+    if (m_streamPosition < startSlipBuffer)
+    {
+      Seek(m_streamPosition,0);
```
Comment (phunkyfish) [inline src/buffers/ClientTimeshift.cpp]: Space after comma

**Comment by phunkyfish** [comment]
Other than the one minor just name the commit similar to the PR and it’s good.

File: src/buffers/ClientTimeshift.cpp (hunk 1 above)
Comment (emveepee) [reply src/buffers/ClientTimeshift.cpp]: Fixed

**Comment by emveepee** [comment]
> 
> 
> Other than the one minor just name the commit similar to the PR and it’s good.

Ok thanks as always.  @sub3  if user feedback is positive this can be merged and tagged.

**Comment by sub3** [comment]
Yep. First bit of feedback was positive, which is a good sign.

**Comment by emveepee** [comment]
@sub3 since 18.7 is out I hope you don't hold off too much longer if you want this in any non binary distro like LibreElec.  There devs don't ask if we have versions pending.

**Comment by sub3** [comment]
Can I safely ignore the jenkins 'this commit cannot be built'? If so, I'll merge it now.

**Comment by emveepee** [comment]
The code seemed fine but since it is reasonable to see CI for macos I made a style change by removing a void in (void) in a header function definition to match the cpp 

https://github.com/kodi-pvr/pvr.nextpvr/pull/126/commits/0a8ed92a4a6efc04f0c980eec00cb87fd283be74#diff-e9db99c7c3cecea4006d71716d9397d6L94

and it is green now.

**Comment by sub3** [comment]
Yep - I noticed that. I've merged it.

**Pull request merged**